### PR TITLE
Implemented toggled sorting order

### PR DESCRIPTION
**Pull request by AbrahmAB**
As discussed with @samdroid-apps . I added a feature of toggled sorting order for list in the Journal. This will allow the list to be sorted in both ways.

### CODE REVIEW

File: src/jarabe/journal/journaltoolbox.py [hunk 1]
```diff
@@ -727,11 +724,20 @@ def __init__(self):
             menu_box.append_item(button)
 
     def __sort_type_changed_cb(self, widget, property_, icon_name):
+        if(self._property == property_):
```
Comment (samdroid-apps) [inline src/jarabe/journal/journaltoolbox.py]: Please run flake8 on this, but I don't think that it is accepted to put the parentheses in the if statement like C.

File: src/jarabe/journal/journaltoolbox.py [hunk 2]
```diff
@@ -727,11 +724,20 @@ def __init__(self):
...
+        else:
+            self._order = Gtk.SortType.ASCENDING
+
         self._property = property_
-        # FIXME: Implement sorting order
-        self._order = Gtk.SortType.ASCENDING
+        if(self._order == Gtk.SortType.ASCENDING):
+            self._order = Gtk.SortType.ASCENDING
+        else:
+            self._order = Gtk.SortType.DESCENDING
```
Comment (samdroid-apps) [inline src/jarabe/journal/journaltoolbox.py]: Why do you need this?   This does nothing.

**Comment by samdroid-apps** [comment]
Awsome, just a few cleaning up things to do.  Also, please send your patch in 1 commit rather than 2.

File: src/jarabe/journal/journaltoolbox.py (hunk 1 above)
Comment (AbrahmAB) [inline src/jarabe/journal/journaltoolbox.py]: Okay I will remove the parentheses.

**Comment by AbrahmAB** [comment]
All done. :)

**Comment by samdroid-apps** [comment]
Thanks, but please run your code through flake8.  You still have some parenthesis around your ifs :(

Also, have a look at the diff from github.  You are still removing a few empty lines unrelated to your patch.  Please don't do this.

**Comment by AbrahmAB** [comment]
I have made necessary changes again and resolved my silly mistakes.

**Pull request merged**